### PR TITLE
fcitx5*: update fcitx5 and its plugins

### PR DESCRIPTION
**Pull request by Vonfry**
###### Motivation for this change
new releases of fcitx5 and its plugins.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/inputmethods/fcitx5/fcitx5-chinese-addons.nix [hunk 1]
```diff
@@ -31,13 +31,13 @@ in
 
 mkDerivation rec {
   pname = "fcitx5-chinese-addons";
-  version = "5.0.10";
+  version = "5.0.11";
 
   src = fetchFromGitHub {
     owner = "fcitx";
     repo = "fcitx5-chinese-addons";
```
Comment (xrelkd) [inline pkgs/tools/inputmethods/fcitx5/fcitx5-chinese-addons.nix:38]:
```suggestion
    repo = pname;
```

File: pkgs/tools/inputmethods/fcitx5/fcitx5-lua.nix [hunk 2]
```diff
@@ -10,13 +10,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fcitx5-lua";
-  version = "5.0.5";
+  version = "5.0.6";
 
   src = fetchFromGitHub {
     owner = "fcitx";
     repo = "fcitx5-lua";
```
Comment (xrelkd) [inline pkgs/tools/inputmethods/fcitx5/fcitx5-lua.nix:17]:
```suggestion
    repo = pname;
```

File: pkgs/tools/inputmethods/fcitx5/fcitx5-qt.nix [hunk 3]
```diff
@@ -12,13 +12,13 @@
 
 mkDerivation rec {
   pname = "fcitx5-qt";
-  version = "5.0.9";
+  version = "5.0.10";
 
   src = fetchFromGitHub {
     owner = "fcitx";
     repo = "fcitx5-qt";
```
Comment (xrelkd) [inline pkgs/tools/inputmethods/fcitx5/fcitx5-qt.nix:19]:
```suggestion
    repo = pname;
```

**Review by bobby285271 — APPROVED** [review]
Result of `nixpkgs-review pr 157954` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.fcitx5-qt</li>
    <li>libsForQt514.fcitx5-qt</li>
  </ul>
</details>
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>fcitx5</li>
    <li>fcitx5-chewing</li>
    <li>fcitx5-chinese-addons</li>
    <li>fcitx5-configtool</li>
    <li>fcitx5-gtk</li>
    <li>fcitx5-hangul</li>
    <li>fcitx5-lua</li>
    <li>fcitx5-m17n</li>
    <li>fcitx5-mozc</li>
    <li>fcitx5-rime</li>
    <li>fcitx5-table-extra</li>
    <li>fcitx5-table-other</li>
    <li>fcitx5-unikey</li>
    <li>fcitx5-with-addons</li>
    <li>libime</li>
    <li>libsForQt5.fcitx5-qt (libsForQt515.fcitx5-qt ,plasma5Packages.fcitx5-qt)</li>
  </ul>
</details>

Change log: [5.0.13](https://groups.google.com/g/fcitx/c/PPYLyVsR5SQ), [5.0.14](https://groups.google.com/g/fcitx/c/L_F3fRsCKzY)

Simplified Chinese input with fcitx5-chinese-addons works for me (will merge this a day later if there are no objections).

**Comment by Vonfry** [comment]
force push: apply suggestions from @xrelkd

**Pull request merged**